mesos/mesos.go

```diff
@@ -121,6 +137,7 @@ func (m *Mesos) GetMetricTypes(cfg plugin.ConfigType) ([]plugin.MetricType, erro
 
 		for _, key := range agent_stats {
 			namespace := core.NewNamespace(pluginVendor, pluginName, "agent").
+				AddStaticElement("task").
```

Comment:
From the [Mesos documentation](http://mesos.apache.org/documentation/latest/architecture/):

> A framework running on top of Mesos consists of two components: a scheduler that registers with the master to be offered resources, and an executor process that is launched on agent nodes to run the framework’s tasks [...]

Without going into too many details, an executor can run one or more tasks. And there wouldn't be a need to call it `framework/executor` since `framework_id` can be added as a tag.